### PR TITLE
Feature/fix permission issue

### PR DESCRIPTION
**Pull request by dahomost**
Hi,
Would you please check this out. Thank you

### CODE REVIEW

**Comment by dahomost** [comment]
@hypery2k 
Would you please review this code changes. thx

**Comment by hypery2k** [comment]
mmmh seems that the travis ci build isn’t working. I try to build a snapshot that people can install it and try it out. Not sure if it solves the problem really. Can’t remember right now if it’s really needed.

File: src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java [hunk 1]
```diff
@@ -117,8 +117,10 @@ public boolean execute(String action, String rawArgs,
                 Log.d(LOG_TAG, "Can Write Settings: " + retVal);
                 if (!retVal && !action.equals("requestWriteSettings") && !action.equals("getWriteSettings")) {
                     //can't write Settings
-                    callback.error("write settings: false");
-                    return false;
+                    // Removed return false, to avoid getting "write settings: false" in some cases.
```
Comment (hypery2k) [inline src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java]: I think would be better to check which platform is running:
```
if API<23 {
           callback.error("write settings: false");
           return false;
} 
```

**Comment by dahomost** [comment]
You absolutely right, I updated the code 

                    // With Android 6.0+/API level>= 23, user can turn on/off permissions as necessary.
                    // Permissions are explicitely granted during the app runtime.
                    if (Build.VERSION.SDK_INT < 23) {
                       // can't write Settings
                       callback.error("write settings: false");
                       return false;
                    }

**Comment by hypery2k** [comment]
thanks for updating

**Comment by dahomost** [comment]
You're Welcome Sir. 
Thank you as well.

**Pull request merged**